views/play.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Choose your weapon, <%= @player_1_name %> </h1>
```

Comment:
Good use of html makes it easy to understand what the page is about and engages the user